product-service/src/functions/getProductsById/handler.ts

```diff
@@ -0,0 +1,28 @@
+import 'source-map-support/register';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+
+import { formatJSONResponse } from '@libs/apiGateway';
+import { middyfy } from '@libs/lambda';
+import { loadProduct } from '../../db';
+
+export const getProductsById = async (
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> => {
+  try {
+    const { id } = event.pathParameters;
+    if (id === undefined) {
```

Comment:
Why do you need this check? I think if we don't provide a product id in the URL, we just go to /products endpoint that returns all products.